app.js

```diff
@@ -1,13 +1,32 @@
 // JavaScript source code
+const { OAuth2Client } = require('google-auth-library');
```

Comment:
As passport-google-oauth is already required at line 13 please delete this from here and if at all in Package.json then do remove from there too.